apps/bfd-shared-test-utils/pom.xml

```diff
@@ -0,0 +1,83 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>gov.cms.bfd</groupId>
+		<artifactId>bfd-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>bfd-shared-test-utils</artifactId>
+	<packaging>jar</packaging>
+
+	<description>
+		Provides test utility code and dependencies that are shared by more than one of the other BFD modules.
+	</description>
+
+	<dependencies>
+		<dependency>
+			<!-- Provides a database schema mechanism. See the DatabaseSchemaManager
+				class. -->
+			<groupId>org.flywaydb</groupId>
+			<artifactId>flyway-core</artifactId>
+			<version>${flyway.version}</version>
+		</dependency>
+		<dependency>
+			<!-- JDBC driver for working with PostgreSQL DBs on Java 8+ (JDBC 4.2). -->
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>${postgresql.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.hsqldb</groupId>
+			<artifactId>hsqldb</artifactId>
+			<version>${hsqldb.version}</version>
+			<!-- Only used by tests, but included in compile scope for use with test
+				utilities in that scope. -->
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<!-- Provides a sane facade for the giant mess of incompatible logging frameworks in
+				 Java. -->
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+		</dependency>
+		<dependency>
+			<!-- An implementation of the JCL API that routes everything through to SLF4J,
+				 instead. -->
+			<groupId>org.slf4j</groupId>
+			<artifactId>jcl-over-slf4j</artifactId>
+		</dependency>
+		<dependency>
+			<!-- An implementation of the Log4J API that routes everything through to SLF4J,
+				 instead. -->
+			<groupId>org.slf4j</groupId>
+			<artifactId>log4j-over-slf4j</artifactId>
+		</dependency>
+		<dependency>
+			<!-- Provides a JUL handler, SLF4JBridgeHandler, that routes everything to SLF4J,
+				 instead. -->
+			<groupId>org.slf4j</groupId>
+			<artifactId>jul-to-slf4j</artifactId>
+		</dependency>
+		<dependency>
+			<!-- Logback is used as the logging target/backend for SLF4J during runtime: all logging
+				 events will be sent to it. -->
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
+		</dependency>
+		<dependency>
+			<!-- Used to provide DB connection pooling. See https://github.com/brettwooldridge/HikariCP
+                    for details. -->
+			<groupId>com.zaxxer</groupId>
+			<artifactId>HikariCP</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.newrelic.telemetry</groupId>
+			<artifactId>dropwizard-metrics-newrelic</artifactId>
+			<version>${newrelic.version}</version>
+		</dependency>
```

Comment:
all of these dependencies seem like they should be coming in via bfd-shared-utils apart from the hsqldb one

Reply:
shared utils isn't a parent of this pom, so it declares its dependencies here. (And shared-utils has a dependency on this module, but isnt a child of this either)